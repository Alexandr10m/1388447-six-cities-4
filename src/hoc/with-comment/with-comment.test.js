import React from "react";
import renderer from "react-test-renderer";
import withComment from "./with-comment.js";
import PropTypes from "prop-types";


const MockComponent = (props) => <div>{props.children}</div>;

MockComponent.propTypes = {
  children: PropTypes.node.isRequired,
};

const MockComponentWrapped = withComment(MockComponent);

describe(`Snapshot of withComment`, () => {
  it(`withComment should render correctly`, () => {
    const tree = renderer
      .create(
          <MockComponentWrapped
            onSendComment={()=>{}}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
