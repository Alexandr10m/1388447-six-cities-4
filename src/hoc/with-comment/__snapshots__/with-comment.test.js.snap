// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of withComment withComment should render correctly 1`] = `<div />`;
